src/util/Fs.cpp

```diff
@@ -442,6 +442,33 @@ checkNoGzipSuffix(std::string const& filename)
     }
 }
 
+std::streamsize
+size(std::ifstream& ifs)
+{
+    assert(ifs.is_open());
+
+    ifs.ignore(std::numeric_limits<std::streamsize>::max());
```

Comment:
why not something like
```
    // get length of file:
    is.seekg (0, is.end);
    auto length = is.tellg();
    is.seekg (0, is.beg);
```

Reply:
According to https://stackoverflow.com/questions/22984956/tellg-function-give-wrong-size-of-file/22986486#22986486 it may not always tell us the number of bytes.

Reply:
Couple things:
 * As we’re using `tellg` to calculate the current position it’s fine (we get a proper percentage)
 * The reason it’s bigger in that article is that it’s binary vs translated content. Doesn’t matter anyways because of the previous point